akka-distributed-data/src/main/scala/akka/cluster/ddata/ORMap.scala

```diff
@@ -186,10 +436,11 @@ final class ORMap[A, B <: ReplicatedData] private[akka] (
   }
 
   override def needPruningFrom(removedNode: UniqueAddress): Boolean = {
-    keys.needPruningFrom(removedNode) || values.exists {
+    val dataNeedsPruning: PartialFunction[(A, B), Boolean] = {
       case (_, data: RemovedNodePruning) ⇒ data.needPruningFrom(removedNode)
       case _                             ⇒ false
     }
+    keys.needPruningFrom(removedNode) || values.exists { dataNeedsPruning }
```

Comment:
sorry, that was a leftover from the short period where I needed also to prune them tombstones. Reverted.